src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -99,10 +111,166 @@ namespace OpenMS
     // update parameters (dummy)
     setParameters_(1, 1);
 
+    if(use_IDs_)
+    {
+      // map string "modified sequence/charge" to all RTs the feature has been observed in the different maps
+      std::unordered_map<String, std::vector<double>> ided_feat_rts;
+      //std::unordered_map<String, std::vector<const typename MapType::FeatureType*>> ided_feats;
+      double minRT = std::numeric_limits<double>::max();
+      for (const auto& map : input_maps)
+      {
+        for (const auto& feat : map)
+        {
+          if (feat.getRT() < minRT) minRT = feat.getRT();
+          const auto& pepIDs = feat.getPeptideIdentifications();
+          if (!pepIDs.empty())
+          {
+            const auto& hits = pepIDs[0].getHits();
+            //TODO sort (lose const) or assume first is best? Especially since we sort later in run_internal_ anyway
```

Comment:
This should be decided before merge

Reply:
I mean having it const would be great but if it is too much hassle we can go with the sorting for now.